### DISCUSSION TITLE
阿里上传，删除，操作后 不实时同步

### DISCUSSION THREAD

**Discussion by iZuoShou** [AlistGo/alist#2734] [Q&A]
阿里上传，删除，操作后 不实时同步到alist阿里
比如在官方阿里云盘网页中存入一个文件，但是打开alist阿里   不会实时的显示出增加的文件？删除也是。

**Comment by ascdev-jia** [comment]
这个问题有没有人也遇到呢？有没有什么解决方案？

**Reply by iZuoShou** [reply]
我是docker方式，升级重启下就行

**Comment by anwen-anyi** [comment]
(可以手动刷新，在你有文件变更的文件夹里面 然后右下角三个点儿点一下能看到刷新按钮，前提是要登录管理员哦)

**Reply by iZuoShou** [reply]
好的，谢谢。

**Reply by Asukaaaaaa** [reply]
感谢回答，右下角三个点真的太难注意到了😥

**Reply by lyy640** [reply]
牛逼，确实顶用

**Reply by hsuromky** [reply]
牛逼，可以了

**Reply by Theodore9411** [reply]
没有发现右下角有三个点啊

**Reply by iZuoShou** [reply]
登录后右下角就有。点击会出来一个长的菜单，选择第一个刷新就行

**Reply by EmmaSamules** [reply]
🐂🍺
